libpay-braintree/src/test/scala/com/wix/pay/braintree/it/BraintreeGatewayWithOAuthIT.scala

```diff
@@ -0,0 +1,46 @@
+package com.wix.pay.braintree.it
+
+import com.braintreegateway.Environment
+import com.wix.pay.PaymentGateway
+import com.wix.pay.braintree._
+import com.wix.pay.creditcard.{CreditCard, YearMonth}
+import com.wix.pay.model.CurrencyAmount
+import org.specs2.mutable.SpecWithJUnit
+
+
+class BraintreeGatewayWithOAuthIT extends SpecWithJUnit {
```

Comment:
Not sure I agree. You may do that as "learning tests", which are later removed.
Or, have tests against Sandbox in a different project, that runs daily/weekly/monthly/yearly/millennially. Failures in this project alerts you that something may have gone wrong, or that there just had been a network issue or downtime on the Sandbox end, you will have to check, but it does not break your "real" project...